src/ast/mod.rs

```diff
@@ -3027,6 +3027,8 @@ pub enum Statement {
         statement: Box<Statement>,
         /// Optional output format of explain
         format: Option<AnalyzeFormat>,
+        /// Postgres style utility options
+        options: Option<Vec<(Ident, Option<String>)>>,
```

Comment:
Currently its unclear looking at the variable name and type signature what this contains, I'm thinking we could do a couple things to improve readability:
- Can we include an example in the doc comment of what the syntax looks like?
- We can also represent the `(Ident, Option<String>)` as a named struct - then the struct and its fields would be representative of the contents

Reply:
Thank you for your review and constructive comments. 
Add named struct `UtilityOptionList` and `UtilityOption`, and the doc comment.